### DISCUSSION TITLE
Using the configurator and CR-Touch

### DISCUSSION THREAD

**Discussion by dreed47** [mriscoc/Ender3V2S1#929] [Ideas]
I want to run the configurator to generate conf files for my specific machine (S1, F4, TJC, CR-Touch, T1 and CVLM) and I want to use ABL and not UBL.  My question is what option do I select for Leveling?  When I select just autolevel I seem to have issues with the machine ignoring the CR-Touch.  Should I select BLT even though it's a CR-Touch?  Or am I missing something else?

**Comment by mriscoc** [comment]
Use BLT for ABL leveling with 3D/BL/CRTouch

**Reply by dreed47** [reply]
thx

**Comment by dreed47** [comment]
A possible enhancement to the configurator to have hover-over info pop-ups to explain options maybe?

**Reply by mriscoc** [reply]
You can open the json files inside of the _leveling folder to know what each item does.
